bot/exts/fun/latex/__init__.py

```diff
@@ -0,0 +1,7 @@
+from bot.bot import Bot
+from bot.exts.fun.latex._latex_cog import Latex
+
+
+def setup(bot: Bot) -> None:
+    """Load the Latex Cog."""
+    bot.add_cog(Latex(bot))
```

Comment:
Do we still need to separate this cog out like this? Seems like it with it being a single file cog now, there is little point it being in it's own folder.

Reply:
Should the cache directory by bot/exts/fun/cache then?

Reply:
Maybe something like `bot/exts/fun/_latex_cache` to make it clear.